server/src/test/java/org/opensearch/index/query/QueryShardContextTests.java

```diff
@@ -307,6 +307,11 @@ public void testFielddataLookupOneFieldManyReferences() throws IOException {
         assertEquals(Arrays.asList(expectedFirstDoc.toString(), expectedSecondDoc.toString()), collect("field", queryShardContext));
     }
 
+    public void testSearchLookupShardId() throws IOException {
+        final var searchLookup = createQueryShardContext("uuid", null, null).lookup();
```

Comment:
If I recall correctly, we're still building the 2.x branch with a minimum Java version of JDK8. I don't think we can use `var` yet for code that we'll want to backport to 2.x, which I think would apply to this change. (I do look forward to when we can release OpenSearch 3.0 and can start using more modern Java features.)